frontend/lib/src/components/shared/BaseButton/BaseButton.tsx

```diff
@@ -66,17 +68,20 @@ function BaseButton({
     ComponentType = StyledHeaderNoPaddingButton
   } else if (kind === BaseButtonKind.ELEMENT_TOOLBAR) {
     ComponentType = StyledElementToolbarButton
+  } else if (kind === BaseButtonKind.BUTTON_GROUP) {
```

Comment:
I'm unsure if it makes sense to call this `BaseButtonKind.BUTTON_GROUP` since for button group,we might actually want to support different styles and not just the borderless icon style as used in the feedback widget. So maybe we call it more specifically something like `FEEDBACK` button. Btw. there is also already a `BORDERLESS_ICON` style, but I assume we cannot reuse this one?

Reply:
I have used the BorderlessIcon for now 👍

Reply:
hmhm although we want to retain
```javascript
  "&:disabled, &:disabled:hover, &:disabled:active": {
    backgroundColor: theme.colors.transparent,
    borderColor: theme.colors.transparent,
    color: theme.colors.gray50,
  },
```
which is not was the `BorderLessIconButton` is doing and this looks funny when the feedback is disabled. I could pass it in the extra style object though. What do you think?

Reply:
I updated the BorderLessIcon button as it does not seem to be used anywhere else 👍